### ISSUE TITLE
Comments are too bright

### ISSUE THREAD

**Issue by d86leader** [ulwlu/elly.vim#13]
This is especially apperent in vimscript, where the `"` symbol is used for comments and also string literals. I've been confused multiple times when reading your colorscheme file =)

![Screenshot_20201213_200924](https://user-images.githubusercontent.com/6142210/102012833-19ec7580-3d7f-11eb-99bf-f1f081a93bdf.png)

I'm using neovim in konsole

**Comment by ulwlu** [comment]
@d86leader 
Thank you for reporting this one as well as line-number.

I see your what you mean makes sense, but `comment` is very big part of this theme. So I'd like to be careful to change this. Currently I couldn't find any good other colors, but would try the commands below in your editor?


```vim
" 1
highlight Comment guifg=#545759
" 1
highlight Comment guifg=#404243
highlight Comment guifg=#545759
```

**Comment by ulwlu** [comment]
Again, I agree with you and it may be a problem, however, I really don't want to break this color matching.....

It still takes more time to dig colors.

**Comment by d86leader** [comment]
I think I like the first one the most, it's quite bright but distinct from code still. The third one is also nice, and the second is too dark again (:
Please, take your time searching

**Comment by ulwlu** [comment]
@d86leader 
Hi, now I updated the master.
Would you update and check when you have a time?

(I believe it's readable while fitting to theme, but my eye maybe gets confused by trying many colors. I would love to hear your opinion)

**Comment by ulwlu** [comment]
Sorry, but I come to think #797E80 would be matching after using new darker one for a day.
So, I'd like you to use below yourself.

```vim
highlight Comment guifg=#545759
```

**Comment by d86leader** [comment]
Hello again. I've tried the one from master, and it's really similar to the one we started this issue with. :)
I understand if you would like it, I personally like `#545759` more.

And in any case, I think i'm not going to use this colorscheme myself. There are a couple of other problems I find with it, which are pretty much a matter of taste. For example, I don't like when highlights change the background color, especially how it looks with hlsearch.
